site/content/overview/faq.md

```diff
@@ -0,0 +1,22 @@
+# Frequently Asked Questions
+
+#### *FAQ: I have an existing .NET application. Can I use the AWS Deploy Tool to deploy it to AWS?*
+Yes, the AWS Deploy Tool can be used to deploy any cloud-native .NET applications to AWS. A cloud-native .NET application is written in .NET with intent to deploy to Linux, not tied to any Windows specific technology such as Windows registry, IIS or MSMQ, and can be deployed on virtualized compute. It cannot be used to deploy .NET Framework, Desktop, Xamarin, or other applications that do not fit the “cloud-native” criteria.
+
+#### *FAQ: How do the AWS Deploy Tool decide what service to choose for deployment?*
+It examines .NET .csproj files and project dependencies, and inspects the code for attribution and Dockerfile presence to figure out the application type and the service best suited to run the application.
```

Comment:
We support F# as well which is a different extension then `.csproj`. Just say the .NET project file.

Reply:
done